agent/lm_agent/backend_utils/models.py

```diff
@@ -41,6 +41,22 @@ class FeatureSchema(BaseModel):
     booked_total: int
 
 
+class FlatFeatureSchema(BaseModel):
```

Comment:
I'm concerned about this being incompatible with the current implementation of the UI, I see CLI was changed on the PR as well. Why not keep the previous schema?

Reply:
I unflattened it. Instead, I build the objects for the nested schema from the flat results from the db query.

Reply:
Resolved in commit 8eb6c76